cypress/integration/sortTableDom.spec.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+Cypress.Commands.add('isSorted', (columnName, clolumnNumber) => {
+  cy.contains(columnName).click();
+
+  cy.get(`tr:nth-child(n) td:nth-child(${clolumnNumber})`).then(($fcolumn) => {
+    const columns = [...$fcolumn].map((column) => column.innerText);
+    const sortedColumns = [...columns].sort((a, b) => a.localeCompare(b));
+    let counter = 0;
+    const copyOfColumns = [];
+    const copyOfsortedColumns = [];
+
+    for (let i = 0; i < columns.length; i++) {
+      copyOfColumns.push(sortedColumns[i].replace('$', '').replace(',', ''));
+      copyOfsortedColumns.push(columns[i].replace('$', '').replace(',', ''));
+    }
+
+    copyOfColumns.sort((a, b) => a - b);
```

Comment:
why are we sorting twice?

Reply:
technically we don't even need to sort, we just need to check that each next value is larger than the previous value.